app/com/github/pedrovgs/dotto/interpreter/actors/ShowMessageActor.scala

```diff
@@ -0,0 +1,42 @@
+package com.github.pedrovgs.dotto.interpreter.actors
+
+import akka.actor.{Actor, Props}
+import com.github.pedrovgs.dotto.core.DottoApp._
+import com.github.pedrovgs.dotto.interpreter.DottoInterpreter
+import com.github.pedrovgs.dotto.interpreter.actors.ShowMessageActor.ShowMessage
+import play.api.Logger
+
+/**
+  * Akka actor used to show messages into the Led previously configured. The Akka actors system by default ensures that
+  * any actor instance will handle messages one by one. Based on this implementation this actor is going to act as a
+  * blocking queue.
+  */
+class ShowMessageActor extends Actor {
+
+  private val interpreter = new DottoInterpreter(context.actorOf(ShowMessageActor.props))
```

Comment:
Use implicits for DI or a Reader monad. No need to manually create the instances.
```
class A
object A {
 implicit def instance : A = new A
}

class B(implicit a : A) {
 a.whatever
}
```
Implicit resolution will apply all implicit functions avalaible in the companion object or imported in the scope.